README.md

```diff
@@ -63,17 +70,31 @@ fi
 ```
 
 If you don't already have a `.git/hooks/pre-commit` file in your `docs` git repo directory, you can copy the existing `pre-commit.sample` file in that directory as `pre-commit` to use the sample template, or create a new `pre-commit` file in that directory with the above content.
-If you create a new file, you must also make it executable with: `chmod 755 /path/to/my/.git/hooks/pre-commit`.
+If you create a new file, you must also make it executable with: `chmod +x /path/to/my/.git/hooks/pre-commit`.
 
-To ensure your markdown is properly formatted, run `make markdowntest`.
+### Check markdown formatting
+
+To ensure your markdown is properly formatted, run:
+
+```console
+make markdowntest docs/**/*`
+```
+
+### Check for broken links
+
+To check locally for broken links, install [`htmltest`](https://github.com/wjdp/htmltest):
+
+```console
+brew install htmltest
+```
 
-To check locally for broken links, install [`htmltest`](https://github.com/wjdp/htmltest) and run `make htmltest`.
+Then, and run `make htmltest`.
 
 ### Lint JS and Markdown files with Prettier on save
 
 1. Install the [Prettier VS Code Extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
-2. Run `npm install` in the docs folder where you have the docs checked out.
-3. Inside VS code, open the _user_ `settings.json` file by pressing `CMD+SHIFT+P` and typing 'settings', and add the following settings to the end of the file:
+1. Run `npm install` in the docs folder where you have the docs checked out.
+1. Inside VS code, open the _user_ `settings.json` file by pressing `CMD+SHIFT+P`, typing 'settings', selecting **Open User Settings (JSON)**, and appending the following settings to the end of the file:
```

Comment:
```suggestion
1. Inside VS code, open the _user_ `settings.json` file: Press `CMD+SHIFT+P`, type 'settings', select **Open User Settings (JSON)**, and append the following settings to the end of the file:
```
[opt]

Reply:
👏 damn the ~torpedoes~ gerunds, full speed ahead!